### DISCUSSION TITLE
Book Mode

### DISCUSSION THREAD

**Discussion by h80r** [lnreader/lnreader#372] [Ideas]
Is it possible to implement a book reading style? I like to read on a tablet, and a book style with page curling transitions would be greatly appreciated.

There are libraries like react-pageflip that implement these page curl components, but I don't know if it would be useful for your implementation.

**Comment by rajarsheechatterjee** [comment]
Are there any libraries in React Native for it?

The app already has a lot of performance issues so animations are not a priority.
